Source/WebCore/rendering/style/StyleRareInheritedData.cpp

```diff
@@ -36,15 +36,17 @@ namespace WebCore {
 
 struct GreaterThanOrSameSizeAsStyleRareInheritedData : public RefCounted<GreaterThanOrSameSizeAsStyleRareInheritedData> {
     float firstFloat;
+    float secondFloat;
     void* styleImage;
```

Comment:
This should be a `Style::ImageOrNone`. This used to be a pointer but it was changed at some point.

There might be other types that are not correct, I didn't check further.

Reply:
I meant `void* styleImage;`.